Automattic/wp-calypso
# Add expiry notice in My Plan page for Jetpack legacy plans

### Changes proposed in this Pull Request

This PR adds an expiry notice in the _My Plan_ section if a legacy Jetpack plan expires soon. Sequel of https://github.com/Automattic/wp-calypso/pull/44913.

### Implementation notes

Updated the README of `Notice` that had an erroneous example.

### Testing instructions

_Note: You can update the expiry date easily from store admin._

- Select a self-hosted Jetpack site with a legacy plan (Personal, Premium, or Professional)
- Visit the _Plans > My Plan_ section and make sure the offer reset flow is enabled
- Check that you see your plan
- If your plan expires in less than 30 days for a monthly term, or 3 months for a yearly term, you should see the expiry date and a notice below the plan navigation
- Verify that you don't see the notice if the offer reset flow is not enabled or if the plan is not expiring soon

### Screenshots

#### Implementation
<img width="741" alt="Screen Shot 2020-08-14 at 11 22 49 AM" src="https://user-images.githubusercontent.com/1620183/90265840-27214400-de21-11ea-8751-b5da02b11bcd.png">

#### Mockups
![myplans](https://user-images.githubusercontent.com/1620183/90265871-31dbd900-de21-11ea-8dff-905c9b431a6e.jpg)
